pkg/apiserver/apiserver.go

```diff
@@ -323,7 +324,14 @@ func installHandlers(c *ExtraConfig, s *genericapiserver.GenericAPIServer) {
 
 		// Install a post start hook to initialize Tiers on start-up
 		s.AddPostStartHook("initialize-tiers", func(context genericapiserver.PostStartHookContext) error {
-			go c.networkPolicyController.InitializeTiers()
+			go func() {
+				ctx, cancel := wait.ContextForChannel(context.StopCh)
+				defer cancel()
+				// context gets cancelled when the server stops.
+				if err := c.networkPolicyController.InitializeTiers(ctx); err != nil {
+					klog.ErrorS(err, "Failed to initialize system Tiers")
+				}
```

Comment:
This is slightly different from the other PRs (for release-2.1 and release-2.0), as `wait.ContextForChannel` used to return a cancel function (in the K8s apiserver library version used in Antrea v1.15).